refaster-support/src/test/java/tech/picnic/errorprone/refaster/matchers/IsEmptyTest.java

```diff
@@ -55,75 +55,80 @@ void matches() {
             "    return new int[][] {{0}};",
             "  }",
             "",
-            "  Random negative5() {",
+            "  int[] negative5() {",
+            "    int i = hashCode();",
+            "    return new int[i];",
+            "  }",
+            "",
+            "  Random negative6() {",
             "    return new Random();",
             "  }",
             "",
-            "  List<Integer> negative6() {",
+            "  List<Integer> negative7() {",
```

Comment:
Yeah, this renaming is a mess. But proposing an alternative is not on today's menu.